app/components/list/model.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const striptags = require('striptags'),
+  { has, isFieldEmpty } = require('../../services/universal/utils'),
+  { render } = require('../../services/universal/styles'),
+  { toSmartText } = require('../../services/universal/sanitize');
+
+module.exports.save = function (uri, data) {
+  const allowedTags = ['strong', 'em', 's', 'a', 'span'];
+
+  data.orderedList ? data.listType = 'ol' : data.listType = 'ul';
```

Comment:
Nice, but this should be the other way around
```suggestion
  data.listType = data.orderedList ? 'ol' : 'ul';
```

Reply:
💪